formspree/forms/models.py

```diff
@@ -132,6 +132,7 @@ def send(self, http_form, referrer):
                 overlimit = True
 
         now = datetime.datetime.utcnow().strftime('%I:%M %p UTC - %d %B %Y')
+        userIsUpgraded = self.owner.upgraded
```

Comment:
the problem is here:

```
  File "formspree/forms/views.py", line 91, in send
    status = form.send(request.form, request.referrer)
  File "formspree/forms/models.py", line 135, in send
    userIsUpgraded = self.owner.upgraded
AttributeError: 'NoneType' object has no attribute 'upgraded'
```

I still think it should work, but let's forget it, because there's another problem that will require changing this code: we shouldn't check if the form owner is upgraded, because that form owner is mostly useless information, it shows who created the form in the dashboard, only that. All other forms (those created automatically by a spontaneous submission) will not have an "owner_id" property and thus no "owner", these forms are tied to (what I called) "controllers" who can see their submissions and power them up for receiving more than 1000 submissions per month solely by registered emails (those emails each user can register in its account). Since more than one user can control the same form, "controllers" is an attribute that resolves to an array of User objects. This is what we are using in other places (at least since https://github.com/asm-products/formspree/commit/ac653653a0a5ae40e00154bacbcdb07ddb49a153 and https://github.com/asm-products/formspree/commit/1a40d1c66d5d2a1ed2a10630f3972257f22d78be) and what we should use here.